packager/packager.js

```diff
@@ -198,11 +203,15 @@ function statusPageMiddleware(req, res, next) {
 }
 
 function getAppMiddleware(options) {
+  var transformerPath = options.transformer;
+  if (!path.isAbsolute(transformerPath)) {
```

Comment:
Oh that's odd. If we have to support older version, we might add

`path.resolve(transformerPath) === path.normalize(transformerPath)`